config/locales/diaspora/en.yml

```diff
@@ -1273,31 +1273,31 @@ en:
       change_email: "Change email"
       new_password: "New password"
       current_password: "Current password"
-      current_password_expl: "the one you sign in with..."
-      character_minimum_expl: "must be at least six characters"
+      current_password_expl: "The one you sign in with..."
+      character_minimum_expl: "Must be at least six characters"
       export_in_progress: 'We are currently processing your data. Please check back in a few moments.'
       last_exported_at: '(Last updated at %{timestamp})'
-      request_export: 'request my profile data'
-      download_export: 'download my profile'
-      request_export_update: 'refresh my profile data'
-      download_photos: "download my photos"
+      request_export: 'Request my profile data'
+      download_export: 'Download my profile'
+      request_export_update: 'Refresh my profile data'
+      download_photos: "Download my photos"
       your_handle: "Your diaspora* ID"
       your_email: "Your email"
       edit_account: "Edit account"
       receive_email_notifications: "Receive email notifications when:"
-      started_sharing: "someone starts sharing with you"
-      someone_reported: "someone sent a report"
-      mentioned: "you are mentioned in a post"
-      liked: "someone likes your post"
-      reshared: "someone reshares your post"
-      comment_on_post: "someone comments on your post"
-      also_commented: "someone comments on a post you've commented on"
-      private_message: "you receive a private message"
+      started_sharing: "Someone starts sharing with you"
+      someone_reported: "Someone sent a report"
+      mentioned: "You are mentioned in a post"
+      liked: "Someone likes your post"
+      reshared: "Someone reshares your post"
+      comment_on_post: "Someone comments on your post"
+      also_commented: "Someone comments on a post you've commented on"
+      private_message: "You receive a private message"
       change: "Change"
       email_awaiting_confirmation: "We have sent you an activation link to %{unconfirmed_email}. Until you follow this link and activate the new address, we will continue to use your original address %{email}."
       stream_preferences: "Stream preferences"
-      show_community_spotlight: "Show Community Spotlight in stream"
-      show_getting_started: 'Show Getting Started hints'
+      show_community_spotlight: "Show community spotlight in stream"
```

Comment:
I'm wondering the same. Does diaspora* have kind of "branded features" like Twitter's Stream, Tweets, ReTweets, Trends, Hashtags, etc.? Twitter considers their prime features as proper nouns and capitalizes them. I think Google's Android style guide also suggests the same.